lnbits/wallets/lnd_grpc_files/router_pb2.py

```diff
@@ -553,7 +553,6 @@
 
 _ROUTER = DESCRIPTOR.services_by_name["Router"]
 if _descriptor._USE_C_DESCRIPTORS == False:
-
     DESCRIPTOR._options = None
```

Comment:
dont touch the lnd grpc files. also some other files only got a blank line removed from some blackking?